buildconfig/CMake/Packaging/launch_mantidplot.sh.in

```diff
@@ -26,4 +26,4 @@ LD_PRELOAD=${LOCAL_PRELOAD} TCMALLOC_RELEASE_RATE=${TCM_RELEASE} \
     TCMALLOC_LARGE_ALLOC_REPORT_THRESHOLD=${TCM_REPORT} \
     PYTHONPATH=${LOCAL_PYTHONPATH} \
     @WRAPPER_PREFIX@$VGLRUN $GDB $INSTALLDIR/bin/@MANTIDPLOT_EXEC@ "$@"@WRAPPER_POSTFIX@ \
-    || PYTHONPATH=${LOCAL_PYTHONPATH} @PYTHON_EXECUTABLE@ @SCRIPTSDIR@/@ERROR_CMD@
+    || PYTHONPATH=${LOCAL_PYTHONPATH} @PYTHON_EXECUTABLE@ @SCRIPTSDIR@/@ERROR_CMD@ --application=mantidplot
```

Comment:
This and `launch_mantidworkbench.sh` seems to be missing the `--exitcode` param, is this intended?

Reply:
I think that should be set from here https://github.com/mantidproject/mantid/blob/master/buildconfig/CMake/LinuxPackageScripts.cmake#L208 .

Reply:
👍 	Didn't see it on the diff. Sorry about that